db_weshare.sql

```diff
@@ -0,0 +1,226 @@
+
+--
+-- Dumping data for table `request`
+--
+
+LOCK TABLES `request` WRITE;
+/*!40000 ALTER TABLE `request` DISABLE KEYS */;
+INSERT INTO `request` VALUES (1,'Besoin de lait',6,"J\'ai besoin de 6L de lait en bouteilles de 1L",'2020-10-20',NULL,NULL,1,NULL,NULL,1),(2,'Cherche une tondeuse à gazon',1,'Besoin d\'une tondeuse pour mon gazon pendant 2 jours','2020-10-20',NULL,NULL,2,NULL,NULL,8),(3,'Urgent : oeufs',1,'J\'ai besoin d\'une boîte de 12 oeufs','2020-10-22',NULL,NULL,3,NULL,NULL,7),(4,'Tissu floral',15,'Je recherche du tissus avec des motifs floraux, pour 15 mètres au total','2020-10-24',NULL,NULL,4,NULL,NULL,3),(5,'4 baguettes',4,'Besoin de 4 baguettes de pain','2020-10-24',NULL,NULL,5,NULL,NULL,8),(6,'Une barre de douche',1,'J\'ai besoin d\'une barre de douche, j\'ai cassé la mienne','2020-10-26',NULL,NULL,6,NULL,NULL,8),(7,'Seau',1,'Cherche un seau en fer','2020-10-25',NULL,NULL,7,NULL,NULL,8);
+/*!40000 ALTER TABLE `request` ENABLE KEYS */;
+UNLOCK TABLES;
+
+--
+-- Table structure for table `item`
+--
+
+DROP TABLE IF EXISTS `item`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!50503 SET character_set_client = utf8mb4 */;
+CREATE TABLE `item` (
+  `id` int unsigned NOT NULL AUTO_INCREMENT,
+  `title` varchar(255) NOT NULL,
+  PRIMARY KEY (`id`)
+) ENGINE=InnoDB AUTO_INCREMENT=4 DEFAULT CHARSET=latin1;
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+--
+-- Dumping data for table `item`
+--
+
+LOCK TABLES `item` WRITE;
+/*!40000 ALTER TABLE `item` DISABLE KEYS */;
+INSERT INTO `item` VALUES (1,'Stuff'),(2,'Doodads'),(3,'this');
+/*!40000 ALTER TABLE `item` ENABLE KEYS */;
+UNLOCK TABLES;
+
+--
+-- Table structure for table `measurement`
+--
+
+DROP TABLE IF EXISTS `measurement`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!50503 SET character_set_client = utf8mb4 */;
+CREATE TABLE `measurement` (
+  `measurement_id` int NOT NULL AUTO_INCREMENT,
+  `measurement_name` varchar(20) NOT NULL,
+  PRIMARY KEY (`measurement_id`)
+) ENGINE=InnoDB AUTO_INCREMENT=9 DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_0900_ai_ci;
+/*!40101 SET character_set_client = @saved_cs_client */;
+
+--
+-- Dumping data for table `measurement`
+--
+
+LOCK TABLES `measurement` WRITE;
+/*!40000 ALTER TABLE `measurement` DISABLE KEYS */;
+INSERT INTO `measurement` VALUES (1,'litre'),(2,'centilitre'),(3,'mètre'),(4,'centimètre'),(5,'gramme'),(6,'kilogramme'),(7,'boîte/paquet'),(8,'autre');
+/*!40000 ALTER TABLE `measurement` ENABLE KEYS */;
+UNLOCK TABLES;
+
+--
+-- Table structure for table `user`
+--
+
+DROP TABLE IF EXISTS `user`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!50503 SET character_set_client = utf8mb4 */;
+CREATE TABLE `user` (
+  `user_id` int NOT NULL AUTO_INCREMENT,
+  `user_civility` varchar(20) NOT NULL,
+  `user_lastname` varchar(50) NOT NULL,
+  `user_firstname` varchar(255) NOT NULL,
+  `user_phone` int NOT NULL,
+  `user_email` varchar(50) NOT NULL,
```

Comment:
It's the same. It's unnecessary to prefix fields by the table name. They already are in the table.